R/corr.R

```diff
@@ -39,8 +39,13 @@
 #'
 #' # Tidying
 #'
-#' When you [`tidy()`][tidy.recipe()] this step, a tibble with column
-#' `terms` (the columns that will be removed) is returned.
+#' When you [`tidy()`][tidy.recipe()] this step, a tibble returned with 2
+#' columns `terms` and `id`:
+#'
+#' \describe{
+#'   \item{terms}{character, the selectors or variables selected}
```

Comment:
```suggestion
#'   \item{terms}{character, the selectors or variables selected to be removed}
```

Unless this documentation was previously wrong.